cornice_swagger/swagger.py

```diff
@@ -506,17 +506,23 @@ def from_schema_mapping(self, schema_mapping):
 
             for field_schema in response_schema.children:
                 location = field_schema.name
+
                 if location == 'body':
                     title = field_schema.__class__.__name__
                     if title == 'body':
                         title = response_schema.__class__.__name__ + 'Body'
                     field_schema.title = title
-
                     response['schema'] = self.definitions.from_schema(field_schema)
+
                 elif location == 'header':
-                    headers = convert_schema(field_schema)
-                    if 'properties' in headers:
-                        response['headers'] = headers['properties']
+                    header_schema = convert_schema(field_schema)
+                    headers = header_schema.get('properties')
+                    if headers:
+                        # Response headers doesn't accept titles
+                        for header in headers.values():
+                            header.pop('title')
```

Comment:
Oh, I understand now. The code that tries to turn Colander schemas into Swagger schemas is ignorant of where in the hierarchy the Colander schema is, and just copies `title` if it's available. I think the tests are fine as they are. I'm not crazy about code that has to delete `title` if it's in the wrong place. Where is `title` legal? Only on the root schema? If that's true, can we remove it from the `converters` package and just put it in the `CorniceSwagger` class?